src/shared/containers/ResourceViewContainer.tsx

```diff
@@ -261,37 +315,88 @@ const ResourceViewContainer: React.FunctionComponent<{
   return (
     <>
       <div className="resource-details">
-        {!!resource && (
-          <Helmet
-            title={`${getResourceLabel(
-              resource
-            )} | ${projectLabel} | ${orgLabel} | Nexus Web`}
-            meta={[
-              {
-                name: 'description',
-                content: getResourceLabel(resource),
-              },
-            ]}
-          />
-        )}
+        {
```

Comment:
I think you don't need these curly braces when you are wrapping stuff in <> </>

Reply:
good spot